lib/bot.js

```diff
@@ -170,15 +170,22 @@ class Bot {
   sendToDiscord(author, channel, text) {
     const discordChannelName = this.invertedMapping[channel.toLowerCase()];
     if (discordChannelName) {
-      // #channel -> channel before retrieving:
-      const discordChannel = this.discord.channels.find('name', discordChannelName.slice(1));
+      // #channel -> channel before retrieving and select only text channels:
+      const discordChannel = this.discord.channels.find(
+        (o) => o.name === discordChannelName.slice(1) && o.type === 'text'
```

Comment:
@dustinlacewell if you don't have time to fix the tests I can do that for you when merging this. Let me know what you think is best :)